Resources/doc/cookbook/custom-repositories.md

```diff
@@ -54,23 +54,3 @@ $repository = $repositoryManager->getRepository('UserBundle:User');
 /** var array of Acme\UserBundle\Entity\User */
 $users = $repository->findWithCustomQuery('bob');
 ```
```

Comment:
do you say empty line at row 57 ? Because there is an empty line at the end of all files in documentation. If you want I'll remove this empty line.